src/wp-includes/functions.php

```diff
@@ -2079,8 +2080,30 @@ function wp_mkdir_p( $target ) {
 		$dir_perms = 0777;
 	}
 
+	/**
+	 * Fires before the directory creation
+	 *
+	 *
+	 * @since 6.5 The action was added.
+	 *
+	 * @param strin $target Full path to attempt to create
+	 * @param int $dir_perms Directory permissions
```

Comment:
```suggestion
	 * Fires before the directory creation.
	 *
	 * @since 6.5.0
	 *
	 * @param string $target Full path to attempt to create.
	 * @param int $dir_perms Directory permissions.
```
- Each sentence in a docblock's summary and parameter/return descriptions should end with a period `.`.
- There was a double line break between the summary and the `@since` tag.
- There was a typo in the type for `$target`, which read `strin` rather than `string`.